core/preference/preference.go

```diff
@@ -50,36 +82,147 @@ type Preference struct {
-		Description:  "Profile picture of the user",
-		Heading:      "Profile",
+		ResourceType: schema.PlatformNamespace,
+		Name:         PreferenceDisableOrgsOnCreate,
+		Title:        "Disable Orgs On Create",
+		Description:  "If selected the new orgs created by members will be disabled by default.This can be used to prevent users from accessing the org until they contact the admin and get it enabled. Default is false.",
+		Heading:      "Platform Settings",
+		SubHeading:   "Manage platform settings and how it's members interact with the platform.",
+		Input:        TraitInputCheckbox,
+		InputHints:   "true,false",
+		Default:      "false",
+	},
+	{
+		ResourceType: schema.PlatformNamespace,
+		Name:         PreferenceDisableOrgsListing,
+		Title:        "Disable Orgs Listing",
+		Description:  "If selected will disallow non-admin APIs to list all organizations on the platform. Default is false.",
+		Heading:      "Platform Settings",
+		SubHeading:   "Manage platform settings and how it's members interact with the platform.",
+		Input:        TraitInputCheckbox,
+		InputHints:   "true,false",
+		Default:      "false",
+	},
+	{
+		ResourceType: schema.PlatformNamespace,
+		Name:         PreferenceDisableUsersListing,
+		Title:        "Disable Users Listing",
+		Description:  "If selected will will disallow non-admin APIs to list all users on the platform. Default is false.",
+		Heading:      "Platform Settings",
+		SubHeading:   "Manage platform settings and how it's members interact with the platform.",
+		Input:        TraitInputCheckbox,
+		InputHints:   "true,false",
+		Default:      "false",
+	},
+	{
+		ResourceType: schema.PlatformNamespace,
+		Name:         PreferenceInviteWithRoles,
+		Title:        "Invite With Roles",
+		Description:  "Allow inviting new members with set of role ids. When the invitation is accepted, the user will be added to the org with the roles specified. This can be a security risk if the user who is inviting is not careful about the roles he is adding and cause permission escalation. Note: this is dangerous and should be used with caution. Default is false.",
+		Heading:      "Platform Settings",
+		SubHeading:   "Manage platform settings and how it's members interact with the platform.",
+		Input:        TraitInputCheckbox,
+		InputHints:   "true,false",
+		Default:      "true",
+	},
+	{
+		ResourceType: schema.PlatformNamespace,
+		Name:         PreferenceInviteMailSubject,
+		Title:        "Invite Mail Subject",
+		Description:  "The subject of the invite mail sent to new members.",
+		Heading:      "Platform Settings",
+		SubHeading:   "Manage platform settings and how it's members interact with the platform.",
+		Input:        TraitInputText,
+		Default:      "You have been invited to join an organization",
+	},
+	{
+		ResourceType: schema.PlatformNamespace,
+		Name:         PreferenceInviteMailBody,
+		Title:        "Invite Mail Body",
+		Description:  "The body of the invite mail sent to new members.",
```

Comment:
We should also add in the description what template variables are available for use. (in mail link and mail otp body as well)

Reply:
I didnt quite get the this? Do we need the variable names like `invite_email_subject` which the user will set. If that the case, it's being printed in with the name field in the response body. Let me know if you mean something else

Reply:
No, go template variables like `{{.Otp}}` and org id, etc